### DISCUSSION TITLE
Apache not running correctly

### DISCUSSION THREAD

**Discussion by Vignesh-Jothi** [leokhoa/laragon#846] [Q&A]
![Screenshot 2024-07-30 092535](https://github.com/user-attachments/assets/5916cb7b-e4fd-403c-bb09-e478d929e534)

I tried to install the vcredist but still not running !?

**Comment by Pen-y-Fan** [comment]
The C++ redistributable version and architecture need to be the same as the version of Apache installed. E.g. If you install the VC14 32-bit version of Apache, you need the VC14 32-bit C++ redistributable.
